docs/meshcloud.metadata-tags.md

```diff
@@ -21,9 +21,9 @@ meshPanel seamlessly integrates any tags that can be set by allowing users to in
 
 ## Tags restricted to Administrators
 
-Administrators can configure tags so that only [Partner users](./administration.index.md) can edit them. These tags are called **restricted tags**. End-users cannot edit restricted tags in meshPanel, but they can view them at anytime. Partner users can view and edit restricted and unrestricted tags.
+Administrators can configure tags so that only [admin users](./administration.index.md) can edit them. These tags are called **restricted tags**. End-users cannot edit restricted tags in meshPanel, but they can view them at anytime. admin users can view and edit restricted and unrestricted tags.
```

Comment:
```suggestion
Administrators can configure tags so that only [admin users](./administration.index.md) can edit them. These tags are called **restricted tags**. End-users cannot edit restricted tags in meshPanel, but they can view them at anytime. Admin users can view and edit restricted and unrestricted tags.
```